### PR TITLE
Suggest using at least pnpm 7.7.1; fix Vercel to use 7.11.0

### PR DESCRIPTION
**Pull request by mmiszy**
pnpm introduced several breaking changes in the lockfile in versions 7.6.0 and 7.7.0.

I've generated this lockfile with 7.7.1 and tested with 7.8.0, 7.9.0 and 7.11.0 – all work fine.
Hence, we suggest using anything above 7.7.0 but pin Vercel to 7.11.0.

### CODE REVIEW

File: apps/checkout/package.json [hunk 1]
```diff
@@ -47,5 +47,5 @@
     "node": ">=16.0.0",
     "pnpm": "^7.5.0"
```
Comment (witoszekdev) [inline apps/checkout/package.json:48]: We should update it just like in main `package.json`:

```suggestion
    "pnpm": ">=7.7.1"
```

File: apps/storefront/package.json [hunk 2]
```diff
@@ -81,5 +81,5 @@
     "node": "^14 || ^16 || ^18",
     "pnpm": "^7.5.0"
```
Comment (witoszekdev) [inline apps/storefront/package.json:82]:
```suggestion
    "pnpm": ">=7.7.1"
```

File: packages/ui-kit/package.json [hunk 3]
```diff
@@ -79,5 +79,5 @@
     "node": ">=16.0.0",
     "pnpm": "^7.5.0"
```
Comment (witoszekdev) [inline packages/ui-kit/package.json:80]:
```suggestion
    "pnpm": ">=7.7.1"
```

**Review by witoszekdev — APPROVED** [review]

**Review by lkostrowski — APPROVED** [review]
Acknowledged :) @zaiste @2can FYI - please check if this changes something in CLI flow.

CLI should not be aware of the implementation details, so hopefully CLI can read pnpm version (if even has to) from package and consume it. I hope we don't have to keep track on such changes in the future

**Pull request merged**